plugins/scheduler-docker-local/scheduler-run

```diff
@@ -80,14 +80,16 @@ scheduler-docker-local-scheduler-run() {
   declare -a DOCKER_START_ARGS_ARRAY
   if [[ "$DOKKU_DETACH_CONTAINER" != "1" ]]; then
     DOCKER_START_ARGS_ARRAY+=("--attach")
+
+    has_tty && DOCKER_START_ARGS_ARRAY+=" --interactive"
   fi
 
   local EARLY_EXIT_CODE=0 EXIT_CODE=0
   if [[ "$DOKKU_DETACH_CONTAINER" == "1" ]]; then
-    "$DOCKER_BIN" container start "${DOCKER_START_ARGS_ARRAY[@]}" "$CONTAINER_ID" >/dev/null || CONTAINER_EXIT_CODE=$?
```

Comment:
@safeforge Why would you remove quotes here? Adding them won't hurt. On the contrary not having them could hurt.

Reply:
@benjamin-thomas When `DOCKER_START_ARGS_ARRAY` contains a multiples arguments the command `docker container start` will be executed as a single arguments, which is wrong.

```
vagrant@dokku:~$ docker container start "--detach-keys --interactive" abc
unknown flag: --detach-keys --interactive
See 'docker container start --help'.
```

At the moment `DOCKER_START_ARGS_ARRAY` contains zero arguments when `DOKKU_DETACH_CONTAINER == 1` and the command is executed without errors. I think is better to remove quotes for a better future integration.

Reply:
Ouch, that's a tricky thing indeed. Thanks for that feedback :)